### DISCUSSION TITLE
Git ignore some Files

### DISCUSSION THREAD

**Discussion by loukasstoltz** [nWidart/laravel-modules#1504] [Q&A] [answered]
Hi, 
I have try to git add the Modules Respository, git track them all, but sometime they are empty (I only have the ./module.json file) ...
Do you have any idea? 

I have check all .gitignore files...
i also have tried to remake a git from scratch... 

Thank you kindly ! 
Loukas

**Comment by dcblogdev** [comment, marked as answer]
If you have folders that are empty they won't be tracked with GIT to keep them you can add a .gitkeep file which will ensure the folder is tracked.
